src/Marten.FSharp/Marten.FSharp.fs

```diff
@@ -378,6 +378,9 @@ module Queryable =
         |> Lambda.ofArity1
         |> q.Count
 
+    let countAsync<'a> (q: IQueryable<'a>) = q.CountAsync()
```

Comment:
`countAsync` -> to `countTask` and see if there's a version that takes a CancellationToken and implmented a `countTaskCt` version.

Reply:
ok, did the things